src/SFA.DAS.IdentifyDataLocks.Web/Pages/_DataLockRow.cshtml.cs

```diff
@@ -18,19 +18,19 @@ public class DataLockRowModel
         public bool IsLocked { get; }
 
         private readonly CollectionPeriod period;
-        private readonly Func<DataMatch, object> valueExtractor;
+        internal Func<DataMatch, object> valueExtractor;
```

Comment:
Let's try GitHub's suggestion feature again.  I've never yet had this result in working code!

I don't think this needs to be made more accessible that it was, does it?

```suggestion
        private readonly Func<DataMatch, object> valueExtractor;
```

Reply:
I was using this in the derived class and then moved away, but didn't change the accessibility here.